### PR TITLE
Add multi arch tests support

### PR DESCRIPTION
**Pull request by williamdes**
### Description

Add multi arch tests support to prevent packaging people to have to patch our test suite because one arch fails to pass tests

Relates to: #16913

### CODE REVIEW

**Comment by mauriciofauth** [comment]
Testing on a 32 bit platform is enough, isn't it?

**Comment by williamdes** [comment]
> Testing on a 32 bit platform is enough, isn't it?

No no, as you can see there is some platforms failing but some not, I think a full test is perfect :)

**Comment by williamdes** [comment]
See: https://github.com/phpmyadmin/phpmyadmin/actions/runs/872055231

**Comment by williamdes** [comment]
My next commit will fix this errors for: `arm32v6`, `arm32v7`, `i386`
All other archs are not affected for some reason
```
There were 3 errors:

1) PhpMyAdmin\Tests\UtilTest::testExtractValueFromFormattedSize with data set #1 ('10GB', 10737418240.0)
TypeError: Argument 2 passed to PhpMyAdmin\Tests\UtilTest::testExtractValueFromFormattedSize() must be of the type int, float given, called in /app/vendor/phpunit/phpunit/src/Framework/TestCase.php on line 1526

/app/test/classes/UtilTest.php:956
/app/vendor/phpunit/phpunit/phpunit:61

2) PhpMyAdmin\Tests\UtilTest::testGetFormattedMaximumUploadSize with data set #5 (2147483648.0, 'MiB', '2,048')
TypeError: Argument 1 passed to PhpMyAdmin\Tests\UtilTest::testGetFormattedMaximumUploadSize() must be of the type int, float given, called in /app/vendor/phpunit/phpunit/src/Framework/TestCase.php on line 1526

/app/test/classes/UtilTest.php:1388
/app/vendor/phpunit/phpunit/phpunit:61

3) PhpMyAdmin\Tests\UtilTest::testGetFormattedMaximumUploadSize with data set #6 (21474836480.0, 'GiB', '20')
TypeError: Argument 1 passed to PhpMyAdmin\Tests\UtilTest::testGetFormattedMaximumUploadSize() must be of the type int, float given, called in /app/vendor/phpunit/phpunit/src/Framework/TestCase.php on line 1526

/app/test/classes/UtilTest.php:1388
/app/vendor/phpunit/phpunit/phpunit:61
```

**Comment by williamdes** [comment]
Ready for review, #16913 will fix the last error

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -9,6 +9,53 @@ on:
...
+          fail-fast: false
+          matrix:
+            include:
+              - { php-version: '7', arch: 'amd64', exclude-phpunit-groups: 'selenium,extension-iconv' }
+              - { php-version: '7', arch: 'arm64v8', exclude-phpunit-groups: 'selenium,extension-iconv' }
+              - { php-version: '7', arch: 'arm32v7', exclude-phpunit-groups: 'selenium,extension-iconv,32bit-incompatible' }
+              - { php-version: '7', arch: 'arm32v6', exclude-phpunit-groups: 'selenium,extension-iconv,32bit-incompatible' }
+              - { php-version: '7', arch: 'i386', exclude-phpunit-groups: 'selenium,extension-iconv,32bit-incompatible' }
+              - { php-version: '7', arch: 'ppc64le', exclude-phpunit-groups: 'selenium,extension-iconv' }
+              - { php-version: '7', arch: 's390x', exclude-phpunit-groups: 'selenium,extension-iconv,32bit-incompatible' }
```
Comment (utkarsh2102) [inline .github/workflows/tests.yml:26]: Thanks a bunch for adding these! 🎉

**Review by utkarsh2102 — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**